qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/search/SearchCustomerInput.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic.portal.portlet.customers.search
+
+/**
+ * Input interface for the {@link life.qbic.portal.portlet.customers.search.SearchCustomer} use case
+ *
+ * This interface describes the methods the use case exposes to its caller.
+ *
+ * @since: 1.0.0
+ *
+ */
+interface SearchCustomerInput {
+
```

Comment:
```suggestion
    /**
    * This method triggers the search for a customer with matching firstname and matching lastname
    *
    * @param firstName: The first name of the customer.
    * @param lastName: The last name of the customer
    * @since: 1.0.0
    */
```